source/user-manual/reference/tools/manage_agents.rst

```diff
@@ -53,9 +53,15 @@ Options
 |               +-----------------------------+-----------------------------------------+
 |               | **Supported installations** | Agent                                   |
 +---------------+-----------------------------+-----------------------------------------+
-| **-F <sec>**  | Remove agents with duplicated IP if disconnected since <sec> seconds. |
+| **-R <sec>**  | Remove agents with duplicate IP if registered since <sec> seconds.    |
```

Comment:
This section looks good to me but I would require also the review of @pereyra-m .

Reply:
The description of the options is correct!

Reply:
Thank you both!